docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc

```diff
@@ -61,6 +61,56 @@ Default URL
 By default the web interface can be accessed here:
 http://localhost:9990/console.
 
+=== Custom HTTP Headers
+
+For the responses returned from the HTTP management interface it is also possible to define custom constant HTTP headers that will be added to any response based on matching a configured prefix against the request path.
+
+As an example it could be desirable to add a HTTP header `X-Help` which points users to the correct location to obtain assistance.  The following management operation can be executed within the CLI to activate returning this header on all requests.
+
+[source,options="nowrap"]
+----
+/] /core-service=management/management-interface=http-interface: \
```

Comment:
`\]` at the beginning of the command looks weird.

Reply:
I can remove that if we want, was just showing the CLI command prompt before the command